src/WebJobs.Script.WebHost/Controllers/KeysController.cs

```diff
@@ -108,11 +108,11 @@ public KeysController(WebScriptHostManager scriptHostManager, ISecretManager sec
 
         [HttpPut]
         [Route("admin/functions/{name}/keys/{keyName}")]
-        public Task<IActionResult> Put(string name, string keyName, Key key) => PutKeyAsync(keyName, key, name, ScriptSecretsType.Function);
+        public Task<IActionResult> Put(string name, string keyName, [FromBody] Key key) => PutKeyAsync(keyName, key, name, ScriptSecretsType.Function);
 
         [HttpPut]
         [Route("admin/host/{keys:regex(^(keys|functionkeys|systemkeys)$)}/{keyName}")]
-        public Task<IActionResult> Put(string keyName, Key key) => PutKeyAsync(keyName, key, GetHostKeyScopeForRequest(), ScriptSecretsType.Host);
```

Comment:
We have integration (scenario) tests that should have been failing with this issue. I'm guessing those are not running, is that right?

Reply:
integration tests are commented out

Reply:
Sent out PR for enabling integration tests:
https://github.com/Azure/azure-webjobs-sdk-script/pull/2361